consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java

```diff
@@ -371,9 +372,18 @@ protected boolean documentsConsistent(JsonNode sourceDocument, JsonNode destinat
         Iterator<Entry<String, JsonNode>> nodeIterator = sourceDocument.fields();
 
         while (nodeIterator.hasNext()) {
-            Entry<String, JsonNode> sourceNode = nodeIterator.next();
-            if(!getJobConfiguration().getComparisonExclusionPaths().contains(sourceNode.getKey())) {
-                if(!sourceNode.getValue().equals(destinationDocument.findValue(sourceNode.getKey()))) {
+            Entry<String, JsonNode> sourceEntry = nodeIterator.next();
+
+            List<String> excludes = getJobConfiguration().getComparisonExclusionPaths();
+            if(excludes == null || !excludes.contains(sourceEntry.getKey())) {
+                JsonNode sourceNode = sourceEntry.getValue();
+                JsonNode destinationNode = destinationDocument.findValue(sourceEntry.getKey());
+
+                if((JsonNodeType.NULL.equals(sourceNode.getNodeType())
+                        && destinationNode == null)){
```

Comment:
Both `sourceNode` and `destinationNode` are instances of `JsonNode` but the null check conditions are different.  Would this be better?

```java
if((sourceNode == null || JsonNodeType.NULL.equals(sourceNode.getNodeType())
      && (destinationNode == null || JsonNodeType.NULL.equals(destinationNode.getNodeType()))) {
```